build.gradle

```diff
@@ -1013,8 +1013,19 @@ project('test:system') {
         commandLine './kubernetes/setupTestPod.sh'
     }
 
+    task runFluentBitUninstall(type: Exec) {
+        environment 'KEEP_PVC', System.getProperty("keepSystemTestLogs", "")
+        commandLine './kubernetes/fluentBitSetup.sh', 'uninstall'
+    }
+
+    task runFluentBitInstall(type: Exec) {
+        dependsOn 'runFluentBitUninstall'
```

Comment:
This actually diverges from the current bevhaviour for other components, i.e. the components deployed for a system test must be explicitly cleaned up before rerunning the `startK8SystemTests` task. Should I remove this to keep with that behaviour?